Dockerfile

```diff
@@ -61,11 +84,21 @@ RUN chown -R $APP_USER:$APP_USER $APP_DATA
 COPY packaging/conf/database.yml ./config/database.yml
 
 # Run the npm postinstall manually after it was copied
-RUN DATABASE_URL=sqlite3:///tmp/db.sqlite3 RAILS_ENV=production bundle exec rake assets:precompile
+RUN DATABASE_URL='nulldb://nohost' RAILS_ENV=production bundle exec rake assets:precompile
 
 # Include pandoc
-RUN DATABASE_URL=sqlite3:///tmp/db.sqlite3 RAILS_ENV=production bundle exec rails runner "puts ::OpenProject::TextFormatting::Formats::Markdown::PandocDownloader.check_or_download!"
+RUN DATABASE_URL='nulldb://nohost' RAILS_ENV=production bundle exec rails runner "puts ::OpenProject::TextFormatting::Formats::Markdown::PandocDownloader.check_or_download!"
+
+# Remove node_modules and entire frontend
+RUN rm -rf $APP_PATH/node_modules/ $APP_PATH/frontend/node_modules/
```

Comment:
I believe this would need to happen in the same RUN instructions than the npm install to have effect on the final size

Reply:
The steps needed would be `npm install && assets precompile && remove node modules` in a single step then? I already thought about using the separate `FROM` images to ensure the node modules are not included in the final image, but since frontend assets are sadly still requring ruby to be properly compiled, this is not (yet) a viable option.

I'll look into what's needed to reduce the image size